Chapter-3-microservice-extraction/Fitnet/Src/Passes/Fitnet.Passes.IntegrationEvents/Fitnet.Passes.IntegrationEvents.csproj

```diff
@@ -1,5 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <ItemGroup>
-    <PackageReference Include="evolutionaryarchitecture.fitnet.common.infrastructure" Version="1.1.5" />
+    <PackageReference Include="MassTransit.RabbitMQ" Version="8.1.1" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\..\Fitnet.Common\Fitnet.Common.Infrastructure\Fitnet.Common.Infrastructure.csproj" />
```

Comment:
Why did you change it to project reference of a relative path instead of 1.1.7 package?

Reply:
That was for local testing. I've reverted.